flasher/overlays/installer/usr/local/bin/installer.sh

```diff
@@ -101,11 +101,25 @@ if [ ${BMAP_EXITCODE} != 0 ] ; then
   read -p "Press return to shutdown your computer..."
   shutdown -h now
 fi
+sync
+
+# wait for target device to ready
+sleep 10
```

Comment:
Hmm... why is this necessary? Shouldn't udevadm be able to wait for everything?

Reply:
The sleep is belt & braces. We can remove that in a followup if things work now. cc @santoshmahto7